src/sveta/Assignment_20_EvenNo.java

```diff
@@ -0,0 +1,49 @@
+package sveta;
+
+public class Assignment_20_EvenNo {
+
+	public void whilePrint() {
+		System.out.println("using While :");
+		int count = 0;
+		int i=1;
+		while(count<5) {
+			if(i%2==0) {
+				System.out.println(i);
+				count++;
+			}
+			i++;
+		}
+	}
+	
+	public void forprint() {
+		System.out.println("using for :");
+		int count=1;
+		for (int i=2;count<=5;i++) {
```

Comment:
We have to print first 5 number,not to restrict it to 5 iteration

Reply:
sorry I resolve it. but In this case my intention is restrict the value that was print five time.
I mean print 5 even no.